src/main/groovy/org/savantbuild/plugin/java/testng/JavaTestNGPlugin.groovy

```diff
@@ -73,6 +75,12 @@ class JavaTestNGPlugin extends BaseGroovyPlugin {
    *   groovyTestNG.test(groups: ["unit", "performance"])
    * </pre>
    *
+   * Supported command line options:
+   *  --onlyFailed    only runs test that failed on previous test run
+   *  --onlyChanges   only run tests for java classes and test classes changed on this branch
+   *    --n=D         only look at last D commits
+   *  --keepXml       keep the testNG xml file
```

Comment:
It is an XML file, but are we asking the plugin to not delete the last test runner definition? 

I'm ok with calling it `XML` if there is nothing else more descriptive, just thought I'd ask. 

If we keep `XML`, it should be `--keepXML` or `--preserveXML` - keeping `XML` uppercase.